workflows/smrnaseq.nf

```diff
@@ -124,26 +125,37 @@ workflow SMRNASEQ {
     // SUBWORKFLOW: Read QC and trim adapters
     //
 
+    if(!params.skip_qc){
     FASTQC_FASTP (
         ch_cat_fastq,
+        ch_fastp_adapters,
         false,
         false
     )
-
     ch_versions = ch_versions.mix(FASTQC_FASTP.out.versions)
-
     reads_for_mirna = FASTQC_FASTP.out.reads
 
+
     //
     // SUBWORKFLOW: mirtrace QC
     //
-    FASTQC_FASTP.out.adapterseq
-    | join( ch_cat_fastq )
-    | map { meta, adapterseq, fastq -> [meta + [adapter:adapterseq], fastq] }
-    | MIRTRACE
+    ch_outputs_for_mirtrace = FASTQC_FASTP.out.adapterseq
+    .join( ch_cat_fastq )
+    .map { meta, adapterseq, fastq -> [meta + [adapter:adapterseq], fastq] }
+    .collect()
```

Comment:
This is not really important but for me, it would be nicer this way:
```suggestion
FASTQC_FASTP.out.adapterseq
    .join( ch_cat_fastq )
    .map { meta, adapterseq, fastq -> [meta + [adapter:adapterseq], fastq] }
    .collect()
    .set { ch_outputs_for_mirtrace }
 ```